_episodes/03-skimage-images.md

```diff
@@ -261,28 +261,30 @@ We will start by reading the image and displaying it.
 import imageio.v3 as iio
 
 # read input image
-image = iio.imread(uri="data/maize-root-cluster.jpg")
+roots = iio.imread(uri="data/maize-root-cluster.jpg")
 
 # display original image
 fig, ax = plt.subplots()
-plt.imshow(image)
+plt.imshow(roots)
 ~~~
 {: .language-python}
 
 
 Now we can threshold the image and display the result.
 
 ~~~
+# copy the image so it can be manipulated
+roots = roots.copy()
 # keep only high-intensity pixels
-image[image < 128] = 0
+roots[roots < 128] = 0
 
 # display modified image
 fig, ax = plt.subplots()
-plt.imshow(image)
+plt.imshow(roots)
 ~~~
 {: .language-python}
 
-The NumPy command to ignore all low-intensity pixels is `image[image < 128] = 0`.
+The NumPy command to ignore all low-intensity pixels is `roots[roots < 128] = 0`.
```

Comment:
I would suggest `maize_roots` instead of `roots` (roots could be confused with the mathematical sense, e.g., roots of an equation).